turbopack/crates/turbopack-ecmascript-runtime/src/asset_context.rs

```diff
@@ -15,6 +15,8 @@ pub fn get_runtime_asset_context(environment: Vc<Environment>) -> Vc<Box<dyn Ass
             enable_typescript_transform: Some(TypescriptTransformOptions::default().cell()),
             ..Default::default()
         },
+        // TODO: Somehow this fails to compile when enabled.
```

Comment:
This is what I was investigating in the first place, the Turbopack runtime doesn't downlevel to the minimum esmodules browsers, i.e. it doesn't pass es2020 when running `npx es-check es2020 test/e2e/app-dir/hello-world/.next/static/**/*.js`. We'll have to figure out why it fails during minification with this enable.